libs/ngxtension/inject-query-params/src/inject-query-params.spec.ts

```diff
@@ -1,12 +1,25 @@
-import { Component } from '@angular/core';
+import { Component, numberAttribute } from '@angular/core';
 import { TestBed } from '@angular/core/testing';
 
 import { provideRouter } from '@angular/router';
 import { RouterTestingHarness } from '@angular/router/testing';
-import { injectQueryParams } from './inject-query-params';
+import { queryParams } from './inject-query-params';
```

Comment:
Minor: I think following the `inject` prefix for CIFs is more intentional/consistent.

Reply:
Works for me. I just thought it was really simple and resembled the angular input signals :)

Reply:
There is a small diff between inputs and this fn. Inputs don't inject anything under the hood, while this function does. And if the inject prefix is not used, devs may think that it's safe to use directly in a non injection context, but that would throw an error. 

So the naming is intentional here.